packages/cspell-lib/config/cspell-default.json

```diff
@@ -24,14 +24,13 @@
     "languageSettings": [
         { "languageId": "*",                                   "dictionaries": ["companies", "softwareTerms", "misc", "filetypes"] },
         { "languageId": "csharp", "allowCompoundWords": true,  "dictionaries": ["csharp", "dotnet", "npm"] },
-        { "languageId": "javascript,javascriptreact,mdx",      "dictionaries": ["typescript", "node", "npm"] },
+        { "languageId": "javascript,javascriptreact",          "dictionaries": ["typescript", "node", "npm"] },
         { "languageId": "typescript,typescriptreact,mdx",      "dictionaries": ["typescript", "node", "npm"] },
-        { "languageId": "javascriptreact,typescriptreact,mdx", "dictionaries": ["html", "css", "fonts"] },
-        { "languageId": "markdown,asciidoc",                   "dictionaries": ["npm", "html"] },
-        { "languageId": "html,pug,jade",                       "dictionaries": ["html", "fonts", "typescript", "css", "npm"] },
-        { "languageId": "handlebars",                          "dictionaries": ["html", "css", "fonts", "typescript"] },
+        { "languageId": "javascriptreact,typescriptreact,mdx", "dictionaries": ["html", "html-symbol-entities", "css", "fonts"] },
+        { "languageId": "markdown,asciidoc",                   "dictionaries": ["npm", "html", "html-symbol-entities"] },
+        { "languageId": "html,pug,jade,php,handlebars",        "dictionaries": ["html", "fonts", "typescript", "css", "npm", "html-symbol-entities"] },
         { "languageId": "json,jsonc",                          "dictionaries": ["node", "npm"] },
-        { "languageId": "php",                                 "dictionaries": ["php", "html", "fonts", "css", "typescript", "npm"] },
+        { "languageId": "php",                                 "dictionaries": ["php"] },
```

Comment:
Since it was mostly copying the HTML and template languages, just added it up there, and added the on unique